source/viz.md

```diff
@@ -1333,25 +1399,27 @@ grid of separate histogram plots.
 ```{index} altair; facet
 ```
 
-We use the `facet` function to create a plot 
+We use the `facet` function to create a plot
 that has multiple subplots arranged in a grid.
-The argument to `facet` specifies the variable(s) used to split the plot 
+The argument to `facet` specifies the variable(s) used to split the plot
 into subplots, and how to split them (i.e., into rows or columns).
-If the plot is to be split horizontally, into rows, 
+If the plot is to be split horizontally, into rows,
 then the `rows` argument is used.
-If the plot is to be split vertically, into columns, 
+If the plot is to be split vertically, into columns,
 then the `columns` argument is used.
-Both the `rows` and `columns` arguments take the column names on which to split the data when creating the subplots. 
+Both the `rows` and `columns` arguments take the column names on which to split the data when creating the subplots.
 
 ```{code-cell} ipython3
 
-morley_hist = alt.Chart(morley_df).mark_bar(opacity = 0.5).encode(
-    x = alt.X("Speed", bin=alt.Bin(maxbins=50)),  
-    y=alt.Y('count()', stack=False),
-    color = "Expt:N").properties(height=100, width=300)
-
-final_plot_facet = (morley_hist + v_line).facet(row = 'Expt:N', data = morley_df)
+morley_hist = alt.Chart(morley_df).mark_bar(opacity=0.5).encode(
+    x=alt.X("Speed", bin=alt.Bin(maxbins=50)),
+    y=alt.Y("count()", stack=False),
+    color="Expt:N"
```

Comment:
This code does not produce the intended plot. The plot is now faceted, but all the facets have the same (multicolor) histogram. Please refer to R version of textbook to see what this should look like.

Reply:
see the following plot -- somehow that one is right